DecodeCx1.py

```diff
@@ -5,195 +5,160 @@
 
 
 tester_details=[]
-r=''  # To save the value of block_insights response
+n1=9  # No. of callings for all blocks

```

Comment:
can we rename it to `number_of_blocks` ?

Reply:
yeah.. sure.